TRANSFORMERS/ARITHMETIC/ADD/ADD.py

```diff
@@ -8,12 +8,14 @@
 def ADD(
     a: OrderedPair | Scalar | Vector, b: list[OrderedPair | Scalar | Vector]
 ) -> OrderedPair | Scalar | Vector:
-    """Add 2 or more numeric arrays, matrices, dataframes, or constants element-wise.
-    When a constant is added to an array or matrix, each element in the array or
-    matrix will be increased by the constant value. If 2 arrays or matrices of different
-    sizes are added, the output will be the size of the larger array or matrix with
-    only the overlapping elements changed.
     """
+    The ADD node adds two or more numeric arrays, matrices, dataframes, or constants element-wise.
+
+    When a constant is added to an array or matrix, each element in the array or matrix will be increased by the constant value. 
+    
+    If two arrays or matrices of different sizes are added, the output will be the size of the larger array or matrix with only the overlapping elements changed.
```

Comment:
Probably good to have an inputs section to describe the inputs (should probably get the node devs to do it though)